src/data/sigma/basic.lean

```diff
@@ -98,6 +98,16 @@ f ⟨x,y⟩
 def sigma.uncurry {γ : Π a, β a → Type*} (f : Π x (y : β x), γ x y) (x : sigma β) : γ x.1 x.2 :=
```

Comment:
(I can't leave a suggestion on the lines containing the docstrings.) Could you refer to `Pi_curry` in the docstrings for `sigma.curry` and `sigma.uncurry`?

Reply:
`equiv.Pi_curry`, you mean?